Autopilot/PathManager/Inc/CommWithAttitudeManager.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Communicates with the attitude manager to send commands and receive useful data.
+ * Author: Anthony Bertnyk
+ */
+
+#ifndef COMM_WITH_ATTITUDE_MANAGER_HPP
+#define COMM_WITH_ATTITUDE_MANAGER_HPP
+
+//Commands to send to attitude manager.
+typedef struct CommandsForAM{
+	float roll,pitch,yaw;	// commanded orientation (radians)
+	float airspeed;			// commanded airspeed (m/s)
+} CommandsForAM;
+
+//Data to receive from attitude manager.
+typedef struct AttitudeData{
```

Comment:
Both these structs will need to be understood by AttitudeManager and PathManager. Could you define them somewhere common to both ? Perhaps just AutoPilot/Inc would be fine.